.travis.yml

```diff
@@ -7,9 +7,6 @@ branches:
   only:
     - master
 cache: false
-before_install:
```

Comment:
We do not need to specify this in our travis configuration because this is specified in the toolkit's [docker-compose](https://github.com/cerner/terra-toolkit/blob/master/src/wdio/docker-compose.yml). So we can actually utilize the same docker build locally and in our Travis CI builds, which reduces the places to specify our build setup.